content/en/community/hackathons/sessions/debugging/content/Tracepoints_Reading-Trace-Data.md

```diff
@@ -38,6 +38,6 @@ This can happen when no tracepoint was ever called.
 
 Any saved trace file can be later processed with the `trace.py` script. In our example:
```

Comment:
I would add the pull path of the `trace.py` script, specifically the fact that the command below is given from the unikraft core directory.